engine/gameobject/src/gameobject/res_gameobject.cpp

```diff
@@ -1,19 +1,20 @@
 // Copyright 2020 The Defold Foundation
```

Comment:
The file was previously named `res_prototype.cpp` but I think `res_gameobject.cpp` is the name we're searching for when looking through our code.